library/validation/validation_script.js.php

```diff
@@ -0,0 +1,174 @@
+<?php
+/*If the validation (new) that uses the validate.js library was set on "on" in globals include the following libraries*/
+if($GLOBALS['new_validate']) {
+?>
+    <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/vendors/moment.js"></script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/vendors/validate/validate.js"></script>
+    <script type="text/javascript" src="<?php echo $GLOBALS['rootdir'] ?>/../library/js/vendors/validate/validate_extend.js"></script>
+<?php
+ }
+?>
+
+<script language='JavaScript'>
+    <?php /*Added 2 parameters to the already existing submitme form*/
+    /*new validate: Use the new validation library (comes from globals)*/
+    /*e: event*/
+    /*form id: used to get the validation rules*/?>
+
+    function submitme(new_validate,e,form_id) {
+        somethingChanged = false;
```

Comment:
Not quite.  You only want to do this if validation is successful and the form will be submitted.  If the submit does not happen then somethingChanged must be left as-is, otherwise there will be no warning if the user then abandons the form.

Also would you put the standard comment header at the top of this file to indicate author, GPL license etc.?